### PR TITLE
Update font asset paths to respect variable when present

### PR DESCRIPTION
**Pull request by AllenBW**
@skateman @epwinchell 

I promise this doesn't do darn thing but help 👍 🏅 

Webpack and other bundlers sometimes need flexible asset paths to compile, this pr makes the resource path amenable to a flexible asset path. 

This is in support of https://github.com/ManageIQ/manageiq-ui-service/issues/850

### CODE REVIEW

File: assets/stylesheets/_font-fabulous.scss [hunk 1]
```diff
@@ -3,23 +3,24 @@
...
 
 @font-face {
   font-family: "font-fabulous";
-  src: font-url(if($ff-asset-helper, "font-fabulous/font-fabulous.eot", "../fonts/font-fabulous/font-fabulous.eot"));
-  src: font-url(if($ff-asset-helper, "font-fabulous/font-fabulous.eot?#iefix", "../fonts/font-fabulous/font-fabulous.eot?#iefix")) format("embedded-opentype"),
-       font-url(if($ff-asset-helper, "font-fabulous/font-fabulous.woff2", "../fonts/font-fabulous/font-fabulous.woff2")) format("woff2"),
-       font-url(if($ff-asset-helper, "font-fabulous/font-fabulous.woff", "../fonts/font-fabulous/font-fabulous.woff")) format("woff"),
-       font-url(if($ff-asset-helper, "font-fabulous/font-fabulous.ttf", "../fonts/font-fabulous/font-fabulous.ttf")) format("truetype"),
-       font-url(if($ff-asset-helper, "font-fabulous/font-fabulous.svg#font-fabulous", "../fonts/font-fabulous/font-fabulous.svg#font-fabulous")) format("svg");
+  src: font-url(if($ff-asset-helper, "font-fabulous/font-fabulous.eot", '#{$ff-font-path}/font-fabulous.eot'));
```
Comment (skateman) [inline assets/stylesheets/_font-fabulous.scss]: We :heart: consistency. Could you please use `"` everywhere?

Comment (AllenBW) [inline assets/stylesheets/_font-fabulous.scss]: Yeah maaaaaaiiii bad,  wasn't sure if it would still obey the variable placement

**Comment by AllenBW** [comment]
oh gosh darn it, killing the other crap

File: assets/stylesheets/_font-fabulous.scss (hunk 1 above)
Comment (skateman) [inline assets/stylesheets/_font-fabulous.scss]: I think the `"` is safer when you're using interpolations as Ruby doesn't have interpolation for `'` and SASS is ruby-ish :stuck_out_tongue:

**Comment by AllenBW** [comment]
@skateman go to sleep 😪

**Pull request merged**